code/lib/manager-api/src/modules/layout.ts

```diff
@@ -283,7 +283,6 @@ export const init: ModuleFn = ({ store, provider, singleStory, fullAPI }) => {
     api,
     state: merge(api.getInitialOptions(), persisted),
     init: () => {
-      api.setOptions(merge(api.getInitialOptions(), persisted));
```

Comment:
Hey @ndelangen, can you weigh in on this change, please? The removed line was [introduced](https://github.com/storybookjs/storybook/pull/18550/files#diff-a8e2c0efe97f49b3d7be7fcd346a40715f169f63bdb513d0e45e51af07f3c221R303) in your PR, per [this comment](https://github.com/storybookjs/storybook/issues/23550#issuecomment-1661064451).

Our understanding is that these options were already set at this point by `url.ts`, and this line clobbers those.

Reply:
@limhenry FYI, the one with most knowledge about this line is on holiday, we will come back to to this next week!